src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -145,25 +187,44 @@ private enum Flags : byte
             if ((isAutoProperty && hasGetAccessor) || hasInitializer)
             {
                 Debug.Assert(!IsIndexer);
-                string fieldName = GeneratedNames.MakeBackingFieldName(_name);
-                BackingField = new SynthesizedBackingFieldSymbol(this,
-                                                                      fieldName,
-                                                                      isReadOnly: (hasGetAccessor && !hasSetAccessor) || isInitOnly,
-                                                                      this.IsStatic,
-                                                                      hasInitializer);
+                // PROTOTYPE(semi-auto-props): Make sure that TestSemiAutoPropertyWithInitializer (when enabled back) is affected by this.
+                // That is, if we removed "hasInitializer", the test should fail, or any other test should get affected.
+                BackingField = CreateBackingField(isCreatedForFieldKeyword: hasInitializer && !isAutoProperty);
+                _lazyBackingFieldSymbolForFieldKeyword = null;
```

Comment:
>_lazyBackingFieldSymbolForFieldKeyword = null; [](http://example.com/codeflow?start=16&length=46)

We are still creating two distinct field symbols? Please read my comment for ```_lazyBackingFieldSymbolForFieldKeyword``` in commit 49.
The ```BackingField``` shouldn't be backed by the ``_lazyBackingFieldSymbolForFieldKeyword``` should be initialized here instead of the ```BackingField```